### PR TITLE
Fix invalid yml and ordering 

### PR DESCRIPTION
**Pull request by mosabua**
For the tools data.

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -572,7 +573,7 @@
     - urltext: Because SQL is everywhere and so is Python from TrinoFest 2023
       url: https://trino.io/blog/2023/07/03/trino-fest-2023-ibis.html
     - urltext: Trino, Ibis, and wrangling Python in the SQL ecosystem from Trino
-    Community Broadcast 49
```
Comment (bitsondatadev) [inline _data/tools.yml:575]: @mosabua is this change intentional?

**Review by bitsondatadev — CHANGES_REQUESTED** [review]

**Review by dain — APPROVED** [review]

**Review by ryandoowan — APPROVED** [review]

File: _data/tools.yml (hunk 1 above)
Comment (mosabua) [reply _data/tools.yml:575]: yeah .. the yml was broken..
